starter-code/css/main.css

```diff
@@ -3,8 +3,56 @@ Colors to use:
 dark blue: #06365f
 light blue: #1c5380
 */
-
+html {
+  box-sizing: border-box;
+}
+*, *:before, *:after {
+  box-sizing: inherit;
+}
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  padding: 0;
+  background-image: url(../images/bgtile.png);
+  background-repeat: repeat-x;
+  
+  
+}
+
+/*-- layout --*/
+/*
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 100px 100px 5px 100px;
+}
+#phone-img {
+  width: 25%;
+}
+#info {
+  width: 40%;
 }
+*/
+header {
+  display: flex;
+  flex-direction: row;
```

Comment:
I see what you are doing here :eyes:. A note, if you do flex, it's direction is row by default. We will see how to avoid the button from expanding like this.